jmix-search/search-flowui/src/main/java/io/jmix/searchflowui/view/result/SearchResultsView.java

```diff
@@ -0,0 +1,331 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.searchflowui.view.result;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableMap;
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.Html;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.virtuallist.VirtualList;
+import com.vaadin.flow.data.renderer.ComponentRenderer;
+import com.vaadin.flow.router.BeforeEnterEvent;
+import com.vaadin.flow.router.Route;
+import io.jmix.core.*;
+import io.jmix.core.metamodel.datatype.Datatype;
+import io.jmix.core.metamodel.datatype.impl.FileRefDatatype;
+import io.jmix.core.metamodel.model.MetaClass;
+import io.jmix.core.metamodel.model.MetaProperty;
+import io.jmix.flowui.DialogWindows;
+import io.jmix.flowui.UiComponents;
+import io.jmix.flowui.ViewNavigators;
+import io.jmix.flowui.kit.component.button.JmixButton;
+import io.jmix.flowui.view.*;
+import io.jmix.flowui.view.navigation.RouteSupport;
+import io.jmix.search.SearchProperties;
+import io.jmix.search.searching.*;
+import io.jmix.searchflowui.component.SearchField;
+import io.jmix.searchflowui.component.SearchFieldContext;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.*;
+
+@Route(value = "search/results", layout = DefaultMainViewParent.class)
+@ViewController("search_SearchResultsView")
+@ViewDescriptor("search-results-view.xml")
+@DialogMode(width = "50em", height = "42.5em", resizable = true)
+public class SearchResultsView extends StandardView {
+
+
+    public static final String QUERY_PARAM_VALUE = "value";
+    public static final String QUERY_PARAM_ENTITIES = "entities";
+    public static final String QUERY_PARAM_STRATEGY = "strategy";
+    public static final String QUERY_PARAM_NEED_RELOAD = "needReload";
+
+    protected static final Map<String, String> systemFieldLabels = ImmutableMap.<String, String>builder()
+            .put("_file_name", "fileName")
+            .put("_content", "content")
+            .build();
+
+    @Autowired
+    protected MessageBundle messageBundle;
+    @Autowired
+    protected UiComponents uiComponents;
+    @Autowired
+    protected DataManager dataManager;
+    @Autowired
+    protected Metadata metadata;
+    @Autowired
+    protected MessageTools messageTools;
+    @Autowired
+    protected ViewNavigators viewNavigators;
+    @Autowired
+    protected IdSerialization idSerialization;
+    @Autowired
+    protected EntitySearcher entitySearcher;
+    @Autowired
+    protected SearchStrategyManager searchStrategyManager;
+
+    @ViewComponent
+    protected VerticalLayout contentBoxWithName;
+
+    protected SearchResult searchResult;
+
+    protected SearchStrategy searchStrategy;
```

Comment:
Format the code.

1 - Constants
2 - ViewComponent
3 - Autowired
4 - Variables

Example - https://github.com/jmix-framework/jmix/blob/master/jmix-quartz/quartz-flowui/src/main/java/io/jmix/quartzflowui/view/trigger/TriggerModelDetailView.java